app/routes/contacts/index.js

```diff
@@ -45,5 +49,17 @@ export default Route.extend({
     if (this.can.cannot('read index')) {
       this.transitionTo('index');
     }
+    /*
+    } else if (this.get('currentUser.role_id') === 'staff_admin') {
+      let self = this;
+      this.prefixes.available().then(function(value) {
+        if (value <= 0) {
+          self.get('flashMessages').danger(self.prefixes.msg_zero);
+        }
+      }, function(reason) {
+        console.debug(reason);
+      });
+    }
+    */
```

Comment:
Should we go ahead and delete this?

Reply:
Yes dont leave dead code. That's what VCS is for.

Reply:
Absolutely, I agree.  I will be more careful!